### DISCUSSION TITLE
Possible to set HEVC via GoPro Labs?

### DISCUSSION THREAD

**Discussion by rdipietro** [gopro/labs#1019] [General]
Hi,

The current GoPro Labs documentation never mentions video encoding (e.g., `Preferences → General → Video Compression → HEVC` on the GoPro itself).

Is changing the encoding to HEVC supported and simply not documented? Or is doing this not yet possible with GoPro Labs?

Thanks!

**Comment by dnewman-gpsw** [comment]
This sounds like a older camera. This is no longer that control on today's camera, it will not be added to Labs command set.  It is also an odd control to need via Labs.  Curious what your need is. This should be a set once control, use the Menu or the app once.

**Comment by rdipietro** [comment]
Thanks very much for the quick response.

My hope is to not have to worry about the state of the camera even if I let it sit for a while with a completely dead battery. GoPro Labs magically lets me reset everything with a simple QR code * except * for HEVC.

I am using a GoPro MAX with recent GoPro Labs firmware. Is this behavior different on newer GoPro MAXes? Is the new behavior to always use HEVC?

**Reply by dnewman-gpsw** [reply]
MAX is already only HEVC for 360.  I think only single lens had that option to switch back. The camera will not forget its settings, even without it battery. There is internal non-volatile memory for that.

**Reply by rdipietro** [reply]
Oh okay, that's great. I thought I remembered it switching back before but it's probably just my memory. Thanks
